peribolos.yaml

```diff
@@ -55,6 +55,8 @@ orgs:
       ci:
         description: ""
         privacy: closed
+        members: []
+        maintainers: []
```

Comment:
Can `repos: []` be added to all teams?

Reply:
Added in https://github.com/uor-framework/.github/pull/4/commits/f5c1d18a85c411e4b949c1bd5582273a10542e9b